packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md

```diff
@@ -0,0 +1,35 @@
+## Guidelines
+
+### General
+
+-   For binary settings, use a [slide toggle](#/styles/form-controls/slide-toggle), [toggle](#/components/FlatSelect), or [checkbox](#/components/Checkbox) instead.
```

Comment:
I'm not sure how a slider could be confused with toggles or checkboxs... (which is not a binary settings form anyway). 
It think the general guideline should focus more about why/when we should Slider over a Numeric Input